crnk-core/src/main/java/io/crnk/core/engine/information/resource/ResourceFieldAccess.java

```diff
@@ -13,14 +15,22 @@
 
 	private final boolean filterable;
 
-	public ResourceFieldAccess(boolean postable, boolean patchable, boolean sortable, boolean filterable) {
+	public ResourceFieldAccess(boolean readable, boolean postable, boolean patchable, boolean sortable, boolean filterable) {
```

Comment:
the constructor is not allowed to change, since it is public API => a second constructor should be created.

Reply:
the old one can be deprecated